lib/cadet_web/plug/check_admin.ex

```diff
@@ -0,0 +1,22 @@
+defmodule CadetWeb.Plug.CheckAdmin do
+  @moduledoc """
+  A plug that checks whether :current_user is an admin
```

Comment:
`A plug` is redundant, we know it's a Plug from the module name

Reply:
A more detailed (but not too verbose) explanation will do, e.g what happens if user is not admin?